src/filters/time/time.constants.ts

```diff
@@ -8,8 +8,46 @@ const TIME = {
     FILLER_WORDS,
 };
 
+const RELATIVE_PREPOSITIONS = '(in|after) ';
+const RELATIVE_POSTPOSTIONS = ' (from now|later|after)';
+const HOURS = '(hours|hour|hrs|hr)';
+const MINUTES = '(minutes|minute|mins|min)';
+const SECONDS = '(seconds|second|secs|sec)';
+const TIME_UNITS = {
+    ANY: `(${HOURS}|${MINUTES}|${SECONDS})`,
+    HOURS: `${HOURS}`,
+    MINUTES: `${MINUTES}`,
+    SECONDS: `${SECONDS}`,
+};
+
+const HALF = '(half an|half a)';
+const QUARTER = '(a quarter of a|a quarter of an)';
```

Comment:
what about "go to work 20 to 5" or "go to work 20 mins to 5" or "go to work 20 mins to 5pm"?

You are gonna tell me I'm scope creeping again lol

Reply:
That is not really relative time though?

Reply:
I guess it was missed out of `Time`

Reply:
I think this should be separate as it will require rewriting the time parser, so it is not a quick change.

Reply:
I can even take it myself but maybe not part of this PR